README.md

```diff
@@ -17,9 +17,17 @@ bower install bedrock-consumer
 Installation of the module followed by a restart of your [bedrock][]
 application is sufficient to make the module available.
 
-<!-- ## How It Works
+## API
 
-TODO: -->
+### handleCredential
 
+Process an Identity containing credentials by verifying the basic credential information and then emitting an event containing the verified credential information and all the data necessary to respond to the request. Credential consumer event listeners will determine how to consume the given credentials and then complete the response to the request via the res/next parameters.
+
+## State of Development
```

Comment:
Don't include "State of Development" or "Future work" for now - I'm actively working on this stuff, so just the stuff above line 26 should be included.